env_support/cmake/custom.cmake

```diff
@@ -43,14 +35,21 @@ endif()
 # Include root and optional parent path of LV_CONF_PATH
 target_include_directories(lvgl SYSTEM PUBLIC ${LVGL_ROOT_DIR} ${LV_CONF_DIR})
 
+# Include /examples folder
```

Comment:
Should remove this comment since below one is for `thorvg`.

Reply:
Added back in test and it works